### PR TITLE
crew add verbose error with missing package dependencies

### PR DESCRIPTION
**Pull request by satmandu**
Works properly:
- [x] x86_64

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=crew_brokendep CREW_TESTING=1 crew update
```

### CODE REVIEW

File: bin/crew [hunk 1]
```diff
@@ -1099,21 +1099,23 @@ end
...
-      check_deps = @pkg.dependencies.map {|k, v| k}
-    end
-    # check all dependencies recursively
-    check_deps.each do |dep|
-      # build unique dependencies list
-      unless @dependencies&.include?(dep) || dep == @pkgName
-        @dependencies << dep
-        search dep, true
-        push_dependencies
+    unless @pkg.nil?
```
Comment (skycocker) [inline bin/crew:1102]: or just `return if @pkg.nil?`
or even `return unless @pkg`

and then the whole block below doesn't have to be indented

File: bin/crew [hunk 2]
```diff
@@ -1147,6 +1149,9 @@ def resolve_dependencies
 
   @dependencies.each do |dep|
     depends = nil
+    unless File.exist?("#{CREW_PACKAGES_PATH}#{dep}.rb")
+      abort"Dependency #{dep} was not found.".lightred
```
Comment (skycocker) [inline bin/crew:1153]: there's a missing whitespace right after `abort`. Does it even work without it? That'd be odd

**Review by skycocker — CHANGES_REQUESTED** [review]

File: bin/crew (hunk 2 above)
Comment (satmandu) [reply bin/crew]: Fixed!

**Review by skycocker — APPROVED** [review]

**Comment by uberhacker** [comment]
Thank you for the approval @skycocker.  Nice to know you are still watching. :)

**Pull request merged**